app/services/dashboard_service.rb

```diff
@@ -39,4 +39,12 @@ def format_memory(used, total)
   def mem_num(val)
     ((val || 0) / 1024.0)
   end
+
+  def get_url(ems_id, attr_url)
+    ems_id + '?display=' + attr_url
```

Comment:
This should be `url_for_only_path(:controller => @controller.controller_name, :action => 'show', :id => ems_id, :display=> attr_url)`

Reply:
I am calling it from `ems_physical_infra_dashboard_controller`, but I want to set the url for `ems_physical_infra_controller`. How should I proceed in this case?

Reply:
Well you can pass `:controller => 'ems_physical_infra'` instead. (But then it doesn't make sense in `DashboardService` since it becomes specific to your specific dashboard.)